tensorflow_addons/image/resampler_ops_test.py

```diff
@@ -110,13 +112,13 @@ def test_op_forward_pass_cpu(self, dtype):
     def test_op_backward_pass_gpu(self, dtype):
```

Comment:
You can removed this function since it's already covered in the `test_op_backward_pass`

Reply:
Oh, yeah, thanks @gabrieldemarmiesse.